.github/workflows/ci.yaml

```
@@ -1,17 +1,26 @@
 name: CI
 
-on: push
+on:
+  push:
+  pull_request:
 
 jobs:
   test:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        node: [ '12', '14' ]
+        node: [ '12', '14', '18' ]
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node }}
       - run: npm install
       - run: npm test
+  lint:
+    runs-on: ubuntu-20.04
```

To be sure to always target an up-to-date system, we can use the following instead:

```suggestion
    runs-on: ubuntu-latest
```

As the current LTS is 22.04 and not 20.04.
This will save some time for keeping this up-to-date in sync everywhere.